spec/features/practice_editor/contact_spec.rb

```diff
@@ -110,5 +110,49 @@ def fill_in_contact_fields
             expect(page).to have_field('Name:', with: nil)
             expect(page).to have_field('Role:', with: nil)
         end
+
+        it 'should allow the user to crop multiple contacts -- new and existing' do
```

Comment:
is this why the tests are failing?

Reply:
Yeah, that's the one. The container is getting a `visibility: hidden` attribute, so it errors out.